samples/apps/github-qna-webapp-react/README.md

```diff
@@ -24,7 +24,12 @@ to get answers about it. The sample highlights how [memory](https://aka.ms/sk/me
 and [embeddings](https://aka.ms/sk/embeddings) work along with the
 [SemanticTextPartitioner](../../../dotnet/src/SemanticKernel/SemanticFunctions/Partitioning/SemanticTextPartitioner.cs)
 when the size of the data is larger than the allowed token limited.
-Each SK function will call Open AI to perform the tasks you ask about.​
+Each SK function will call Open AI to perform the tasks you ask about.
+
+This sample will create embeddings only for markdown files by default in
+order to save time and costs.  You can change the filter by going to
+[GitHubSkill.cs](/semantic-kernel/blob/memory-sample/samples/dotnet/github-skills/GitHubSkill.cs)
+and editing the searchPattern filter.
```

Comment:
it's not that simple, e.g. the code can index only text files, if the pattern is `*.pdf` the solution won't work. Also the solution hasn't been tested with code, e.g. if the filter is `*.py`. I would drop this second sentence for now.